src/lando/api/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.mark.django_db(transaction=True)
+def test__views__git2hgCommitMapView(commit_maps, authenticated_client, monkeypatch):
+    mock_catch_up = mock.MagicMock()
+    monkeypatch.setattr("lando.api.views.CommitMap.catch_up", mock_catch_up)
+    response = authenticated_client.get(f"/api/git2hg/test_git_repo/{'a' * 40}")
+    assert response.status_code == 200
+    assert response.json() == {"git_hash": "a" * 40, "hg_hash": "b" * 40}
+
+
+@pytest.mark.django_db(transaction=True)
+def test__views__hg2gitCommitMapView(commit_maps, authenticated_client, monkeypatch):
+    mock_catch_up = mock.MagicMock()
+    monkeypatch.setattr("lando.api.views.CommitMap.catch_up", mock_catch_up)
+    response = authenticated_client.get(f"/api/hg2git/test_git_repo/{'b' * 40}")
```

Comment:
Could we reference a known good SHA from `commit_maps` here instead of hard-coding the SHA?